modules/Cargo.toml

```diff
@@ -36,6 +36,7 @@ clock = ["tendermint/clock", "time/std"]
 # This feature grants access to development-time mocking libraries, such as `MockContext` or `MockHeader`.
 # Depends on the `testgen` suite for generating Tendermint light blocks.
 mocks = ["tendermint-testgen", "clock", "std", "sp-io", "sp-io/std", "sha3", "ripemd"]
+ics11-beefy = ["tendermint-testgen", "clock", "std", "sp-io", "sp-core", "sp-std", "sp-io/std", "sha3", "ripemd"]
```

Comment:
`tendermint-testgen` should probably not be here, it's not a `no-std` compatible crate and `sp-io/std` should also not be enabled.